### DISCUSSION TITLE
Обход блокировки IP

### DISCUSSION THREAD

**Discussion by asgardxf** [bol-van/zapret#223] [Q&A]
Пока не смог запустить у себя скрипт проверки, но есть смысл этим заниматься, если адреса не пингуются? идёт 100% потеря данных, например для адреса bbc.com - 151.101.0.81. Если смысла нет пытаться что-то сделать, возможно нужно в скрипт установки добавить проверку такого рода. Могу реквест сделать, если идея хорошая

**Comment by bol-van** [comment]
вроде уже отвечал в закрытом иссуе

**Comment by NerdzzyDev** [comment]
> Пока не смог запустить у себя скрипт проверки, но есть смысл этим заниматься, если адреса не пингуются? идёт 100% потеря данных, например для адреса bbc.com - 151.101.0.81. Если смысла нет пытаться что-то сделать, возможно нужно в скрипт установки добавить проверку такого рода. Могу реквест сделать, если идея хорошая

---
Попробовал написать инструкцию для таких случаев:
https://github.com/bol-van/zapret/discussions/338
